Auto_Restart_Launcher.bat

```diff
@@ -0,0 +1,116 @@
+@echo off
+:://////////////////////////////////////////////////////////////////////////
+:://           BOIII Dedicated Server Configuration start-up file        ///
+:://////////////////////////////////////////////////////////////////////////
+:://                 Name of the server to identify your bat             ///
+:://////////////////////////////////////////////////////////////////////////
+
+set name=BOIII Nameofserver
+
+:://////////////////////////////////////////////////////////////////////////
+:://      The regex search for the window name of the sever. (optional)   //
+:://                                                                      //
+:://     This will help identify which window needs closing in case the   //
+:://     server errors out or freezes according to filename on g_log      //
+:://                                                                      //
+:://     If hosting multiable. I would suggest putting your hostname      //
```

Comment:
shouldn't it be "multiple"?
"If hosting multiple servers i suggest putting your hostname.."